### PR TITLE
Portfolio - Jessika Hage

### PR DESCRIPTION
**Pull request by jessika-hage**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:title" content="Jessika Hage's Portfolio">
+    <meta property="og:description" content="Here is where you can find all of my work, skills and contact information!">
```
Comment (KarolCislowski) [inline index.html:8]: Nice that you provided OGs here 😸

File: index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/37c653cfef.js" crossorigin="anonymous"></script>
+    <title>Jessika Hages Portfolio</title>
+</head>
+<body>
+
+    <!-- Header -->
+
+    <main class="header">
+        <img class="header-image" src="assets/greg-rosenke-S5YCDN46Wc8-unsplash.png" alt="Pens">
```
Comment (KarolCislowski) [inline index.html]: You used very big photos in your projects (this one is 5004x2815 and 1.9MB).  and it is loading too long what will make a very bad first impression. compress your pictures in photoshop or use apps like https://tinypng.com/

File: index.html [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+    <!-- Section with my projects, big one -->
+    <section class="projects-container">
+
+        <!-- Section with the featured projects -->
+        <h2 class="section-header">FEATURED PROJECTS</h2>
+        <section class="featured-projects">
+            <article class="project-card">
+                <a href="https://github.com/jessika-hage/project-guess-who"  target="_blank" class="project-link" rel="noopener noreferrer">
+                    <img src="assets/guesswho.PNG" alt="Guess Who Game" class="project-image">
```
Comment (KarolCislowski) [inline index.html]: The name of the project needs to be located on top of your picture. 🎨

**Review by KarolCislowski — APPROVED** [review]
It is a very nice portfolio page 🦄  

I left below a shortlist of fixes needed to fit it better to the guidelines. Please make them soon 😸

File: index.html (hunk 2 above)
Comment (jessika-hage) [reply index.html]: I uploaded them on tinypng so hopefully it will be better now!

File: index.html (hunk 3 above)
Comment (jessika-hage) [reply index.html]: Oh, can't believe I missed that. But fixed it now :D Thanks!

**Pull request closed without merging**